src/pages/PlanPage/subRoutes/CurrentOrgPlan/InfoMessageCancellation/InfoMessageCancellation.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react'
+import { z } from 'zod'
+
+import { SubscriptionDetailSchema } from 'services/account'
+
+import InfoMessageCancellation from './InfoMessageCancellation'
+
+const subscriptionDetail = {
+  currentPeriodEnd: 1606851492,
+  cancelAtPeriodEnd: true,
+  defaultPaymentMethod: {
+    card: {
+      brand: 'visa',
+      expMonth: 12,
+      expYear: 2023,
+      last4: '1234',
+    },
+  },
+} as z.infer<typeof SubscriptionDetailSchema>
+
+const wrapper = (subDetail = subscriptionDetail) =>
+  render(
+    <InfoMessageCancellation
+      subscriptionDetail={subDetail}
+      // @ts-expect-error
+      provider="gh"
+      owner="codecov"
+    />
+  )
```

Comment:
Sorry if I wasn't 100% clear beforehand, `render` should be called inside the `it` blocks directly, and stuff in wrapper should be things like query providers, and router items.

Reply:
All good! I gotcha now, fixed in [12127c8](https://github.com/codecov/gazebo/pull/2642/commits/12127c80d350bb5dd23b784304b42964bd3a1c55)